### PR TITLE
Update guide to deploy edge

### PR DESCRIPTION
**Pull request by jogonzal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,13 +42,14 @@ $ npm run build -- --<arg>
...
+Every store is different and you'll have to look at our internal guide to understand how to ship an update. In general, shipping in chrome/edge is easy and shipping in FF/Safari is hard. We don't ship any updates to IE anymore.
 
 #### Edge
 [Instructions here](https://docs.microsoft.com/en-us/microsoft-edge/extensions/guides/packaging/using-manifoldjs-to-package-extensions)
-1. Build locally `npm run build`
+1. Build locally `npm run build:prod`
 2. Run `npm run pack-edge`. The results will be in /OneNoteWebClipper/EdgeExtension
-3. Now follow our internal instructions to update the package
+3. The file you will upload is the `target/edge/OneNoteWebClipper/edgeextension/package/edgeExtension.appx` file. You can also test by uploading the `target/edge/onenotewebclipper/edgeextension/manifest/extension` folder locally in edge
+3. Now follow our internal instructions to update the package.
```
Comment (mttwc) [inline README.md]: Should be 4.

**Review by mttwc — APPROVED** [review]
